examples/sync/src/lib.rs

```diff
@@ -0,0 +1,122 @@
+//! [Any](commonware_storage::adb::any::Any) database sync example library.
```

Comment:
Would just use `[commonware_storage::adb::any::Any]` direct

Reply:
For consistency, start this line with `Synchronize state between a server and client.` (matchcing `Cargo.toml`)